app/main.py

```diff
@@ -2,0 +1,22 @@
+import os
+from typing import Any
+
+
+def move_file(command: str) -> Any:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError
+
+    word_in = parts[1]
+    word_out = parts[2]
+
+    if word_out[-1] == "/":
+        word_out += word_in.split("/")[-1]
+    directories = "/".join(word_out.split("/")[:-1])
```

Comment:
Use `os.path.join` instead of manually joining paths with `/`. This will ensure your code is cross-platform and works on different operating systems.